firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -66,6 +67,7 @@
   private final IidStore iidStore;
```

Comment:
How about changing the name of the existing threadpool?
private final ExecutorService backgroundExecutor;

Reply:
Done.